pandas/core/nanops.py

```diff
@@ -200,12 +200,84 @@ def _get_fill_value(dtype, fill_value=None, fill_value_typ=None):
                 return tslibs.iNaT
 
 
+def _maybe_get_mask(values, skipna, mask, invert=False):
```

Comment:
do we *really* need invert? this is just confusing

Reply:
pls type annotate the parameters

Reply:
`invert` was necessary in `nanmedian()`, but the median calculation is so slow there's minimal impact, so reverting that bit back and removing `invert`